PageObjectPattern/src/test/java/serenityCucumberStepDefinitions/LoginStepDefinitions.java

```diff
@@ -0,0 +1,40 @@
+package serenityCucumberStepDefinitions;
+
+import io.cucumber.java.en.*;
+import net.thucydides.core.annotations.Steps;
+import serenity.pages.LoginPage;
+import serenity.pages.MyAccountPage;
+
+//Cucumber glue code, using regex
+public class LoginStepDefinitions {
+
+    @Steps
+    LoginPage loginPage;
+
+    @Steps
+    MyAccountPage userPage;
+
+    @Given("^Unauthenticated User is on Login Page$")
```

Comment:
Cucumber will understand you step if you just use:
`Unauthenticated User is on Login Page`

Reply:
got it, thanks